lib/nerves_hub/client.ex

```diff
@@ -0,0 +1,47 @@
+defmodule NervesHub.Client do
+  @moduledoc """
+  Responsible for determining if an update should be applied.
+  """
+  require Logger
+
+  @typedoc "Update that comes over a socket."
+  @type update_data :: map()
+
+  @callback update_available(update_data) :: :apply | :ignore | {:reschedule, pos_integer()}
```

Comment:
Could you add some `@doc`'s for the callbacks?

Reply:
:+1:  Done